### ISSUE TITLE
Enhancement: Add button to convert uploaded image to single 32x32 tile

### ISSUE THREAD

**Issue by JeffreyNordlieTR** [FranciscoBiaso/TileMapEditorRealmz#12]
Add support for converting a single uploaded image into a 32x32 tile. This would save the user the hassle of using a separate tool to shrink it themselves.

**Comment by FranciscoBiaso** [comment]
It is possible to convert a single image, have you tested a tool in full mode?

**Comment by JeffreyNordlieTR** [comment]
Yeah I uploaded the image in full mode and it split the image into many 32x32 tiles. I didn't see a way to have the full image itself as one tile. Is there a way to do that?

**Comment by FranciscoBiaso** [comment]
Yes, there is a way. It is not documented yet. You can drag and drop a image from source image or you can hit double left click over the image.

**Comment by JeffreyNordlieTR** [comment]
Ok. I'm a bit confused by the imagine upload GUI especially when it comes to the little tile layout buttons. Mouseover tooltips and documentation would be greatly appreciated!

**Comment by FranciscoBiaso** [comment]
It is not possible to convert an image to 32x32, do a resize operation or scale but is possible to clone or copy a rectangle area 32x32 from a big image. Is that what you had in mind?

**Comment by JeffreyNordlieTR** [comment]
I want to be able to convert a single image into a single tile without resizing it to 32x32 in a separate application.
